core/cog-launcher.h

```diff
@@ -21,21 +21,20 @@ G_BEGIN_DECLS
 
 typedef struct _CogRequestHandler CogRequestHandler;
 
-#define COG_TYPE_LAUNCHER (cog_launcher_get_type ())
+#define COG_TYPE_LAUNCHER (cog_launcher_get_type())
```

Comment:
This file shouldn't be changed at all, so this needs to be removed from this commit.